src/fare-contracts/details/OrderDetails.tsx

```diff
@@ -15,6 +16,11 @@ export const OrderDetails = ({fareContract}: {fareContract: FareContract}) => {
     FareContractTexts.details.orderId(fareContract.orderId),
   );
   const firstTravelRight = fareContract.travelRights[0];
+  const priceString =
+    fareContract.totalAmount &&
+    fareContract.totalAmount !== '' &&
+    formatDecimalNumber(parseFloat(fareContract.totalAmount), language, 2);
```

Comment:
When `totalAmount` is an empty string, this will return the empty string.
You can e.g. use a ternary operator instead with `undefined` in the else case.

Reply:
I tested with empty string, and this is the result, it doesn't show as empty string as shown below. Did I miss something here? 🤔 

<img width="300" src="https://github.com/AtB-AS/mittatb-app/assets/1777333/9ef30c86-f60e-46e0-a245-d156040085da" />

<img width="300" src="https://github.com/AtB-AS/mittatb-app/assets/1777333/c71f3d9c-c0a7-4c26-a4ec-51e549424452" />

Reply:
An empty string is falsy, so it still works as is. However just wanted to note that the `priceString` variable will be an empty string rather than undefined when `totalAmount === ''` with this code. Strictly speaking I think it is better that the `priceString` is explicitly undefined rather than empty, but mainly I mentioned this just to be aware of how the chaining of `&&` works here.

Reply:
I see, then I'll update it to prevent the falsy values, since it might be confusing in the future.